.gitignore

```diff
@@ -1,9 +1 @@
-.env
-.cache
-__pycache__
-/dist
-/build
-.vscode
-*.bak
-*~
-
+ideas.txt
```

Comment:
¿Podéis incluir en el PR solo lo que sea relevante para la corrección del objetivo, en lugar de todo lo que hayáis cambiado?
Entre objetivo y objetivo se pueden hacer ajustes y organizar, pero ¿cómo podríais evitar sobrecargar a la persona encargada de la revisión con esos cambios?
¿Has considerado hacer los ajustes necesarios en la rama main y luego hacer un rebase para que en este PR solo aparezcan los ficheros pertinentes para la revisión?

Reply:
Buenas, cierto, pero el profesor creo que no permite hacer ningun cambio en main hasta que no ha sido revisado por él, puesto que la rama main esta destinada al trabajo realizado y revisado.

Reply:
Puedes hacer cambios en main y luego un rebase.